client/extensions/woocommerce/state/data-layer/ui/products/test/index.js

```diff
@@ -355,38 +371,25 @@ describe( 'handlers', () => {
 
 		it( 'should create variations for a new product', () => {
 			const product1 = { id: { index: 0 }, name: 'Product #1', type: 'variable' };
-			const variationBlack = { id: { index: 4 }, attributes: [ { name: 'Color', options: 'Black' } ] };
 
-			const rootState = {
-				extensions: {
-					woocommerce: {
-						ui: {
-							products: {
-								123: {
-									edits: {
-										creates: [
-											product1,
-										]
-									},
-									variations: {
-										edits: [
-											{
-												productId: { index: 0 },
-												creates: [
-													variationBlack,
-												]
-											}
-										]
-									}
-								},
-							},
-						}
-					}
-				}
+			const productEdits = {
+				creates: [
+					product1,
+				],
 			};
 
-			const productEdits = rootState.extensions.woocommerce.ui.products[ 123 ].edits;
-			const variationEdits = rootState.extensions.woocommerce.ui.products[ 123 ].variations.edits;
+			const variationEdits = [
+				{
+					productId: { index: 0 },
+					creates: [
+						variationBlack,
```

Comment:
Shouldn't we have a version of `variationBlack` with a placeholder ID as well for correctness? Right now this will have a variation in creates during the test that will have a real ID rather than a placeholder one. This applies in a few places.